flask_security/core.py

```diff
@@ -177,6 +181,10 @@
         _('Email not provided'), 'error'),
     'INVALID_EMAIL_ADDRESS': (
         _('Invalid email address'), 'error'),
+    'USERNAME_NOT_PROVIDED': (
+        _('Username not provided'), 'error'),
```

Comment:
If we decide to be more generic the `Username` should be configurable `%(identifier_name)s`.